.travis.yml

```diff
@@ -146,6 +162,27 @@ install:
     - if [[ "$CI_BUILD" == "TRUE" ]]; then ./scripts/buildAmici.sh; fi
     - if [[ "$CI_ARCHIVE" == "TRUE" ]]; then ./scripts/installAmiciArchive.sh; fi
     - if [[ "$CI_PYTHON" == "TRUE" ]]; then ./scripts/installAmiciSource.sh; fi
+    #- if [[ "$TRAVIS_OS_NAME" == "windows" ]]; then pip install --user -U pip; fi
+    # allow PowerShell to run scripts
+    - if [[ "$TRAVIS_OS_NAME" == "windows" ]]; then powershell -Command Set-ExecutionPolicy Unrestricted -Force; fi
+    # Enable Windows developer mode to support symlinks
+    - if [[ "$TRAVIS_OS_NAME" == "windows" ]]; then powershell -Command New-ItemProperty -Path "HKLM:\SOFTWARE\Microsoft\Windows\CurrentVersion\AppModelUnlock" -Name AllowDevelopmentWithoutDevLicense -Value 1  -PropertyType DWord; fi
+    - if [[ "$TRAVIS_OS_NAME" == "windows" ]]; then git clone -c core.symlinks=true https://github.com/ICB-DCM/AMICI.git && cd AMICI; fi
+    - if [[ "$TRAVIS_OS_NAME" == "windows" ]]; then pip -v install python-libsbml; fi
+    - if [[ "$TRAVIS_OS_NAME" == "windows" ]] && [[ "$TRAVIS_PULL_REQUEST" == "false" ]]; then git checkout -qf $TRAVIS_COMMIT; fi
+    - if [[ "$TRAVIS_OS_NAME" == "windows" ]] && [[ "$TRAVIS_PULL_REQUEST" != "false" ]]; then git fetch origin pull/$TRAVIS_PULL_REQUEST/head:$TRAVIS_BRANCH && git checkout $TRAVIS_BRANCH; fi
+    # - if [[ "$TRAVIS_OS_NAME" == "windows" ]]; then powershell -Command Write-Output $BASE_DIR; fi # check content
```

Comment:
remove

Reply:
Here, I assume the I should remove just lined 174, yes, that is test code that should be cleaned up.

Reply:
Who should resolve the conversation? I assume the initiator.

Reply:
No clear rules here, I think. If it is doubtlessly sufficiently addressed, you can resolve, I'd say.